10617 Removing High Priority for calendaring

zacharybrown204 opened the pull request.

zacharybrown204 committed:
10617

Removing prioritization logic from UI and API

zacharybrown204 committed:
10617

Removed some unneeded unit tests
Removed a missed call of unsetHighPriority

zacharybrown204 committed:
10617

Updating unit tests
Removing case high priority from more locations

zacharybrown204 committed:
10617

More updates to case high priority unit tests.

zacharybrown204 committed:
10617

Fixing additional tests after removing high priority.

zacharybrown204 committed:
10617

Removing unit test that should have been testing for high priority.

zacharybrown204 committed:
Merge branch '10617' into 10617-to-test

Mwindo reviewed (commented):
Thanks @zacharybrown204! This is looking good! I wonder if the following are needed?

- https://github.com/ustaxcourt/ef-cms/blob/f55e0a2b168fe9fc274562ec88356d25461900f6/web-client/src/presenter/computeds/formattedEligibleCasesHelper.ts#L17 and related
- https://github.com/ustaxcourt/ef-cms/blob/f55e0a2b168fe9fc274562ec88356d25461900f6/web-api/src/persistence/postgres/cases/schema.ts#L32-L33 (note this is only the schema in app code; it is not making actual changes to the database)

Also, it might be good to write a comment pointing to a followup PR that creates a migration (you can use `npm run migration:generate:postgres`) to drop the columns from the database.

zachrog commented:
> Thanks @zacharybrown204! This is looking good! I wonder if the following are needed?
> 
> * https://github.com/ustaxcourt/ef-cms/blob/f55e0a2b168fe9fc274562ec88356d25461900f6/web-client/src/presenter/computeds/formattedEligibleCasesHelper.ts#L17
>    and related
> * https://github.com/ustaxcourt/ef-cms/blob/f55e0a2b168fe9fc274562ec88356d25461900f6/web-api/src/persistence/postgres/cases/schema.ts#L32-L33
>    (note this is only the schema in app code; it is not making actual changes to the database)
> 
> Also, it might be good to write a comment pointing to a followup PR that creates a migration (you can use `npm run migration:generate:postgres`) to drop the columns from the database.

I agree with Chris. I just want to make sure there is a follow up PR dropping these columns.

Nice job Zach!

zacharybrown204 committed:
10617

Removing a missed instance where we use the case high priority property.

zacharybrown204 committed:
Merge branch '10617' into 10617-to-test

zacharybrown204 commented:
Good catch on the spot in formattedEligibleCaseHandler I had missed that one. Just pushed a commit to handle that one.
I would prefer take out the schema change when I make the migration removing it. It might be a preference but I really don't like when a schema definition and migrations don't match.

I don't have the PR to remove the DB portion yet but I did create a devex ticket to follow up on this. https://github.com/flexion/ef-cms/issues/10712

zacharybrown204 committed:
10617

Editing some unit tests about sorting cases to no longer reference high priority cases.

zacharybrown204 committed:
Merge branch '10617' into 10617-to-test

zacharybrown204 committed:
Merge remote-tracking branch 'origin/test' into 10617-to-test
